core/src/generator.ts

```diff
@@ -0,0 +1,79 @@
+import type { Overrides } from '@jackfranklin/test-data-bot';
+import type { TGeneratorOptions, TGeneratorResult } from './types';
+
+import { build } from '@jackfranklin/test-data-bot';
+
+// /* TYPES DECLARATIONS FROM @jackfranklin/test-data-bot */
```

Comment:
I opened a PR to export the types: https://github.com/jackfranklin/test-data-bot/pull/344